File: app-quality-insights/ide/testData/projects/simpleApp/app/src/main/java/com/example/myapplication/MyActivity.java
package com.example.myapplication;

import android.app.Activity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
public class MyActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_my);

        throw new RuntimeException("oh no");
    }
}

File: android/testData/projects/kotlinWithVersionCatalog/app/src/main/java/google/simpleapplication/MyActivity.kt
package google.simpleapplication

import android.app.Activity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem

class MyActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_my)
    }
}

